tests/test_analytics.py

```diff
@@ -168,3 +169,36 @@ def test_collect_event(self):
         # It's counted as a sitewide event, but not as a library event.
         eq_(3, sitewide_provider.count)
         eq_(1, library_provider.count)
+
+    def test_initialize(self):
+
+        local_analytics = get_one(
+            self._db, ExternalIntegration,
+            protocol=LocalAnalyticsProvider.__module__,
+            goal=ExternalIntegration.ANALYTICS_GOAL
+        )
+
+        # There shouldn't exist a local analytics service.
+        eq_(None, local_analytics)
+
+        # So when the Local Analytics provider is initialized, it will
+        # create one with the default name of "Local Analytics".
+        local_analytics = LocalAnalyticsProvider.initialize(self._db)
+
+        assert isinstance(local_analytics, ExternalIntegration)
+        eq_(local_analytics.name, LocalAnalyticsProvider.NAME)
+
+        self._db.delete(local_analytics)
```

Comment:
I don't think this line does anything (possibly because the delete isn't flushed to the database), because if local_analytics was really deleted, a new one would be created and `local_analytics_2` would have a different ID. I'd just take this line out.